src/test/java/com/google/cloud/spanner/jdbc/MultiplexedSessionsMockServerTest.java

```diff
@@ -205,8 +205,6 @@ public void testUsesMultiplexedSessionInCombinationWithSessionPoolOptions() thro
     CreateSessionRequest request = mockSpanner.getRequestsOfType(CreateSessionRequest.class).get(0);
     assertTrue(request.getSession().getMultiplexed());
     // There should be no regular sessions in use.
-    // However, the query that detects the dialect that is used, uses a regular session.
-    // This should be fixed in the Java client.
-    assertEquals(1, mockSpanner.countRequestsOfType(BatchCreateSessionsRequest.class));
+    assertEquals(0, mockSpanner.countRequestsOfType(BatchCreateSessionsRequest.class));
```

Comment:
The dialect is now also determined using the multiplexed session, meaning that `BatchCreateSessions` won't be called at all in this case.